README.md

```diff
@@ -53,6 +53,123 @@ See [RADIUS.md](RADIUS.md) for a list of supported Attrbiutes.
 
 Many thanks to [On Waves](https://www.on-waves.com/) for sponsoring the RADIUS Authentication implementation.
 
+Example of configuration **RADIUS**:
+```erlang
+%% ...
+{ergw_aaa, [
+    {handlers, [
+        {ergw_aaa_static, [
+            {'Node-Id',        <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-Identifier', <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-IP-Address', {127,0,0,3}},                %% <- CHANGE
+            {'Acct-Interim-Interval',   1800},              %% <- CHANGE
+            {'Framed-Protocol',         'PPP'},
+            {'Service-Type',            'Framed-User'}
+        ]},
+        {ergw_aaa_radius, [
+            {server,
+                {{127,0,0,4}, 1813, <<"CHANGE-ME-SECRET">>} %% <- CHANGE IP and SECRET
+            }
+        ]}
+    ]},
+    {services, [
+        {'Default', [
+            {handler, 'ergw_aaa_static'}
+        ]},
+        {'RADIUS-Acct', [
+            {handler, 'ergw_aaa_radius'}
+        ]}
+    ]},
+    {apps, [
+        {default, [
+            {session, ['Default']},
+            {procedures, [
+                {authenticate, []},
+                {authorize, []},
+                {start, ['RADIUS-Acct']},
+                {interim, ['RADIUS-Acct']},
+                {stop, ['RADIUS-Acct']},
+                {{gx, 'CCR-Initial'},   []},
+                {{gx, 'CCR-Update'},    []},
+                {{gx, 'CCR-Terminate'}, []}
+            ]}
+        ]}
+    ]}
+]},
+%% ...
+```
+Example of configuration **epc-ocs** `function` of **DIAMETER**:
+```erlang
+%% ...
+{ergw_aaa, [
+%% ...
+    {functions, [
+        {'epc-ocs', [
+            {handler, ergw_aaa_diameter},
+            {'Origin-Host', <<"CHANGE-ME">>},                           %% <- CHANGE
+            {'Origin-Realm', <<"CHANGE-ME">>},                          %% <- CHANGE
+            {transports, [
+                [
+                    {connect_to, <<"aaa://CHANGE-ME;transport=tcp">>},  %% <- CHANGE
+                    {recbuf,131072},                                    %% <- CHANGE
+                    {sndbuf,131072}                                     %% <- CHANGE
+                ]
+            ]}
+        ]}
+    ]},
+%% ...
+]},
+%% ...
+```
+Example of configuration **ergw-pgw-epc-rf** `function` of **DIAMETER**:
+```erlang
+%% ...
+{ergw_aaa, [
+    %% ...
+    {functions, [
+        {'ergw-pgw-epc-rf', [
+            {handler, ergw_aaa_diameter},
+            {'Origin-Host', <<"CHANGE-ME">>},                           %% <- CHANGE
+            {'Origin-Realm', <<"CHANGE-ME">>},                          %% <- CHANGE
+            {transports, [
+                [
+                    {connect_to, <<"aaa://CHANGE-ME;transport=tcp">>},  %% <- CHANGE
+                    {recbuf,131072},{reuseaddr,false},{sndbuf,131072},  %% <- CHANGE
+                    {unordered,false}                                   %% <- CHANGE
```

Comment:
unordered is only relevant for SCTP and it is actually prefered that SCTP uses `{unordered, true}` (which is the default).